docs/azure_arc.md

```diff
@@ -0,0 +1,48 @@
+# Deploying to Azure Arc
+
+This template supports deployment to Azure Arc. The following document lists the requirements to setup Azure Arc to support the template, as well as how you deploy the application to an Arc-enable Kubernetes Cluster.
+
+- [Deploying to Azure Arc](#deploying-to-azure-arc)
+  - [Prerequisites](#prerequisites)
+  - [Regions and resource group support](#regions-and-resource-group-support)
+    - [Azure Arc enabled App Service](#azure-arc-enabled-app-service)
+
+## Prerequisites
+
+In order to deploy this template to Azure Arc, you need to have the Arc-enabled Kubernetes Cluster created. This template assumes this has already been done. To learn more about configuring an Arc environment for App Service, please see this [blog](https://aka.ms/ArcEnabledAppServices-Build2021-Blog).
+
+It's also requred to have Arc-enabled dataservices Data Controller installed in the Arc Cluster, please see more [here](https://docs.microsoft.com/en-us/azure/azure-arc/data/create-data-controller). This repository assumes a directly connected data controller.
+
+## Regions and resource group support
+
+[Please check here for latest information](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster) 
+
+### Azure Arc enabled App Service and Data Services
+
+Following these guidelines will deploy the webapi to an ARC-enabled Kubernetes cluster.
+
+To deploy from local environment run the deployment script
+
+```
+./deploy/scripts/install.sh \
+   --resource-name-prefix <resource group name> \
+   --environment-tag <name tag for all resources> \
+   --resource-group-tag <tag> \
+   --location <location (eastus or westeurope)> \
+   --node-env development \
+   --kube-environment-id <kubeEnvironmentId> \
+   --custom-location-id <customLocationId> \
+   --data-controller-id <dataControlleId>
+```
+
+To deploy using the GitHub Actions Workflow, the following is needed in the [config.yaml](../deploy/config.yaml) file:
+
+```
+AZURE_LOCATION: "northeurope"       #Location of Azure hosted resources, e.g. Azure Monitor
+RESOURCE_NAME_PREFIX: "nodewebapi"  #Resource name prefix
+ENVIRONMENT_TAG: "test"             #Resource tag
+WEBAPI_NODE_ENV: "development"      #nodeEnv parameter
+KUBE_ENVIRONMENT_ID: ""             #kubeEnvironmentId to host the webapi on Arc
+CUSTOM_LOCATION_ID: ""              #customLocationId for the kubeEnvironment
+DATA_CONTROLLER_ID: ""              #dataControllerId
```

Comment:
Where does the user find this info?

Reply:
Good callout. They are found on the individual resources (KubeEnvironment, CustomLocation and DataController), these are the resources in ARM, which are tied to the Arc-enabled K8s cluster. I've added some guidance above.

Reply:
What CLI or script commands can they run to get this info?  I'm looking for ways to help them get the info quickly

Reply:
Think of this similar to finding a subscription Id, if you didn't create it you have to ask someone. At least they need to get the customlocation resource Id from whomever created that. This is really a Kubernetes namespace on a Kubernetes cluster somewhere, if you installed it, you know the customer location and can retrieve it's Id. - or do 'az customlocation show -g <RG> -n <name>'.
Based on that they can get the other Ids using the graph (my new favorite thing :-)):
'az graph query -q "where (type =~ 'Microsoft.Web/kubeEnvironments' or type =~ 'microsoft.azurearcdata/datacontrollers') and extendedLocation.name == '<CustomLocation Resource Id>' | where subscriptionId == '<GUID>' | project name, id"'